tests/e2e/e2e.go

```diff
@@ -335,6 +335,12 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 	config.QPS = 20
 	config.Burst = 50
 	kubeCli, _ := kubernetes.NewForConfig(config)
+	if !ginkgo.CurrentGinkgoTestDescription().Failed {
```

Comment:
This will not be called if there are any cases failed in other Ginkgo nodes rather than node-1, but it may not a problem.

Reply:
emmm, any suggestion?

Reply:
😂 No, but after we upgrade Ginkgo to v2, we may get the global test summary. ref https://github.com/onsi/ginkgo/issues/361#issuecomment-814203240